src/features/BI-1982.feature

```diff
@@ -0,0 +1,22 @@
+    Feature: Display an error when creating a duplicate breeding method
+
+    @BI-1982
+	Scenario: Display an error when creating a duplicate breeding method
+    	Given user logs in as "Cucumber Breeder"
+		When user selects "Snacks" on program-selection page
+		And user selects "Program Administration" in navigation
+		And user selects "Breeding Methods" tab
+		When user selects "Create Breeding Method" button
+        Then user can see 'Breeding Methods' tab in Program Management page
```

Comment:
Are lines 6-9 indented for some reason? If there's no semantic reason, I wouldn't indent them.